frontend/src/controllers/notify.ts

```diff
@@ -50,7 +54,7 @@ export class NotifyController implements ReactiveController {
 
   toast(detail: NotifyEvent["detail"]) {
     this.host.dispatchEvent(
-      new CustomEvent("notify", {
+      <NotifyEvent>new CustomEvent(NOTIFY_EVENT_NAME, {
```

Comment:
If we unwrap `NotifyEvent` from `CustomEvent` and use it like this, we'll get better type suggestions — as is, this creates a `CustomEvent<any>` and then coerces it into a `NotifyEvent`, whereas this would actually show errors in the `detail` prop at the point of instantiation
```suggestion
      new CustomEvent<NotifyEvent>(NOTIFY_EVENT_NAME, {
```